drivers/src/hmac384.rs

```diff
@@ -182,6 +172,28 @@ impl Hmac384 {
         Ok(op)
     }
 
+    /// Generate an LFSR seed.
```

Comment:
Can you add a comment about where the seed gets stored after this function is called?

Reply:
Done.